### PR TITLE
Updated timex dependency

### PR DESCRIPTION
**Pull request by andreaswillems**
Thank you for creating verk / verk_web.
I updated the timex dependency to silence build warnings I have when using verk_web. Also I enabled using a local version of bower instead of a global version.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -38,16 +38,18 @@ defmodule VerkWeb.Mixfile do
       {:phoenix_pubsub, "~> 2.0"},
       {:phoenix_html, "~> 2.6"},
       {:gettext, "~> 0.16"},
-      {:verk, "~> 1.7.1"},
+      # {:verk, "~> 1.7.1"},
+      {:verk, github: "andreaswillems/verk", tag: "1.8.0"},
```
Comment (edgurgel) [inline mix.exs:42]: Hey! Do we need a new Verk release as well?

**Comment by edgurgel** [comment]
Thanks for the PR! Will definitely have a closer look later this week

**Comment by andreaswillems** [comment]
sorry, my latest change should not have been merged to your repo.

File: mix.exs (hunk 1 above)
Comment (andreaswillems) [reply mix.exs:42]: For testing and performance purposes, I replaced Jason with Jsonrs in Verk and somehow my change now appears in this PR. I close until I have figured out a clean way to contribute.

**Comment by edgurgel** [comment]
No worries! Thanks anyway!

**Pull request closed without merging**